apps/test/util/testUtils.js

```diff
@@ -279,6 +279,30 @@ exports.createMouseEvent = function mouseEvent(type, clientX, clientY) {
 };
 
 /**
+ * Creates a key event of the given type with the additional parameters
+ * @param {string} type (keydown, keyup, keypress)
+ * @param {bool} keyCodeArg
+ * @param {bool} metaKeyArg
+ * @param {bool} altKeyArg
+ * @param {bool} shiftKeyArg
+ * @param {bool} ctrlKeyArg
+ */
+exports.createKeyEvent = function keyEvent(type, keyCodeArg, metaKeyArg, altKeyArg, shiftKeyArg, ctrlKeyArg) {
+  // Need to use generic "Event" instead of "KeyboardEvent" because of
+  // http://stackoverflow.com/questions/961532/firing-a-keyboard-event-in-javascript#comment-44022523
+  var keyboardEvent = document.createEvent('Event');
+  keyboardEvent.initEvent(type, true, true);
```

Comment:
`initEvent` appears to be deprecated: https://developer.mozilla.org/en-US/docs/Web/API/Event/initEvent 

It looks like you should be able to specify the same behavior using the `Event()` constructor: https://developer.mozilla.org/en-US/docs/Web/API/Event/Event